### DISCUSSION TITLE
beginner

### DISCUSSION THREAD

**Discussion by dontbeacunt1** [Cyfrin/foundry-full-course-cu#2854] [General]
just started learning all about solidity and smart contracts and I still find even the github repo confusing to use. but I hope not to give up and leaving this message here in the discussions tab for future me

**Comment by EngrPips** [comment]
Please don't give up, and always reach out here for help whenever you get stuck. We promise to do our best to help you always.

**Comment by LordAdhaar** [comment]
@dontbeacunt1 good name btw

If you find the github repo difficult to use just as I did use this resource.
https://www.theodinproject.com/lessons/foundations-introduction-to-git
Do the introduction to git and git basics, just two lessons. And all your doubts regarding git and github will be cleared.
Trust.

**Reply by EngrPips** [reply]
Nice heads-up
